No matches found: "sentry:sourcemaps:*"

Run nr sentry:sourcemaps
  nr sentry:sourcemaps
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    SENTRY_AUTH_TOKEN: ***
    SENTRY_ORG: ***
    SENTRY_PROJECT: ***
    SENTRY_DSN: ***
    SENTRY_RELEASE: 68ed3684425c9f867267d293128447530c0c24ad
No matches found: "sentry:sourcemaps:*"
Error: Process completed with exit code 1.

--- a/package.json
+++ b/package.json
@@ -65,7 +65,7 @@
     "lint:fix": "run-s \"lint:fix:biome\" lint:type-check",
     "lint:fix:biome": "biome check --apply-unsafe .",
     "lint:fix:tsc": "run-s \"lint:fix:biome\" lint:type-check:tsc",
-    "sentry:sourcemaps": "run-s sentry:sourcemaps:*",
+    "sentry:sourcemaps": "run-s sentry:sourcemaps:electron sentry:sourcemaps:vite",
     "sentry:sourcemaps:electron": "sentry-cli sourcemaps inject --org $SENTRY_ORG --project $SENTRY_PROJECT --release $SENTRY_RELEASE ./main && sentry-cli sourcemaps upload --org $SENTRY_ORG --project $SENTRY_PROJECT --release $SENTRY_RELEASE ./main",
     "sentry:sourcemaps:vite": "sentry-cli sourcemaps inject --org $SENTRY_ORG --project $SENTRY_PROJECT --release $SENTRY_RELEASE ./src/out && sentry-cli sourcemaps upload --org $SENTRY_ORG --project $SENTRY_PROJECT --release $SENTRY_RELEASE ./src/out",
     "knip": "knip --config knip.config.ts"

